libs/application/templates/financial-aid/src/fields/SummaryForm/SummaryForm.tsx

```diff
@@ -0,0 +1,264 @@
+import * as m from '../../lib/messages'
+import SummaryBlock from '../SummaryBlock/SummaryBlock'
+import { ApproveOptions, FAFieldBaseProps } from '../../lib/types'
+import {
+  Employment,
+  estimatedBreakDown,
+  getNextPeriod,
+  HomeCircumstances,
+} from '@island.is/financial-aid/shared/lib'
+import {
+  getMessageHomeCircumstances,
+  getMessageEmploymentStatus,
+  getMessageApproveOptions,
+  getMessageApproveOptionsForIncome,
+} from '../../lib/formatters'
+import { Breakdown } from '@island.is/financial-aid/shared/components'
+
+const SummaryForm = ({ application, goToScreen }: FAFieldBaseProps) => {
+  const { formatMessage } = useIntl()
+  const { answers, externalData } = application
+
+  const { setValue } = useFormContext()
+
+  const formCommentId = 'formComment'
+
+  return (
+    <>
+      <Box
+        display="flex"
+        alignItems="center"
+        flexWrap="wrap"
+        marginTop={[2, 2, 4]}
+      >
+        <Box marginRight={1}>
+          <Text as="h3" variant="h3">
+            {formatMessage(m.summaryForm.general.descriptionTitle)}
+          </Text>
+        </Box>
+
+        <Text variant="small">
+          {formatMessage(m.summaryForm.general.descriptionSubtitle, {
+            nextMonth: getNextPeriod.month,
+          })}
+        </Text>
+      </Box>
+
+      <Box marginTop={2}>
+        <DescriptionText text={m.summaryForm.general.description} />
+      </Box>
+
+      <Box>{/* TODO estimated aid */}</Box>
+
+      <Box marginTop={2}>
+        <DescriptionText text={m.summaryForm.general.calculationsOverview} />
+      </Box>
+
+      <Box paddingY={[4, 4, 5]} borderTopWidth="standard" borderColor="blue300">
+        <GridRow marginBottom={3}>
+          <GridColumn span={['12/12', '12/12', '12/12', '5/12']}>
+            <Box>
+              <Text fontWeight="semiBold">
+                {formatMessage(m.summaryForm.userInfo.name)}
+              </Text>
+              <Text>{externalData?.nationalRegistry?.data?.fullName}</Text>
+            </Box>
+          </GridColumn>
+          <GridColumn span={['12/12', '12/12', '12/12', '5/12']}>
+            <Box marginTop={[3, 3, 3, 0]}>
+              <Text fontWeight="semiBold">
+                {formatMessage(m.summaryForm.userInfo.nationalId)}
+              </Text>
+              <Text>{externalData?.nationalRegistry?.data?.nationalId}</Text>
+            </Box>
+          </GridColumn>
+          <GridColumn span={['12/12', '12/12', '12/12', '5/12']}>
+            <Box marginTop={3}>
+              <Text fontWeight="semiBold">
+                {formatMessage(m.summaryForm.userInfo.address)}
+              </Text>
+              <Text>{externalData?.nationalRegistry?.data?.address}</Text>
+            </Box>
+          </GridColumn>
+        </GridRow>
+      </Box>
+
+      <SummaryBlock
+        sectionTitle={formatMessage(m.inRelationship.general.sectionTitle)}
+        editAction={() => goToScreen?.('inRelationship')}
```

Comment:
We could change the interface of `SummaryBlock` to take in just an routePath which is a string. Then we can create an enum which are all the screens and then this could be: 

` <SummaryBlock
         sectionTitle={formatMessage(m.inRelationship.general.sectionTitle)}
         routePath={routes.inRelationship}`